### PR TITLE
Avg Pool time

### PR DESCRIPTION
**Pull request by inthedepthsofitall**
#22426

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by inthedepthsofitall** [comment]
#22427 #22427

File: ivy/functional/frontends/paddle/nn/functional/pooling.py [hunk 1]
```diff
@@ -69,3 +69,88 @@ def avg_pool1d(
...
+print("Input Tensor:")
+print(input_tensor)
+
+print("\nAverage Pooling Output:")
+print(avg_pool_output)
+
+print("\nMax Pooling Output:")
+print(max_pool_output)
+
+#lets se how this works
```
Comment (AnnaTz) [inline ivy/functional/frontends/paddle/nn/functional/pooling.py:156]: Hi @inthedepthsofitall, please remove any code except the actual `max_pool1d` and `max_pool2d` functions.

Comment (AnnaTz) [inline ivy/functional/frontends/paddle/nn/functional/pooling.py:156]: Also, you will need to add tests for these new functions. Our documentation describes how to do so and you can also look at the existing tests in our codebase a reference.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

**Comment by AnnaTz** [comment]
Are there corresponding ToDo issues for these functions? Please link your PR to them. Thanks!

**Comment by inthedepthsofitall** [comment]
Hi Anna, I created my own issue as I did not see an average of the pooling
function. I also am not sure what I am getting rid of, asxI added to the
pooling file, unless you are referring that I must create a file called
avg.py.

Also I did create tests for the avg and ran them, but I I will take a look
to see if there was something I did wrong.

**Pull request closed without merging**